.github/workflows/build.yml

```diff
@@ -53,6 +57,7 @@ jobs:
       - name: Check platforms buildx supports
         run: docker buildx inspect --bootstrap
       - name: Build docker image
+        if: github.event_name != 'pull_request'
```

Comment:
What is the purpose of this workflow to be triggered in PRs if we exclude the build step from it?